modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -346,19 +346,17 @@ By disabling P-states and C-states at the pod level, you can configure high prio
 
 .Configuration for high priority workloads
 [cols="1,2", options="header"]
-|====
-|Annotation
-|Description
+|===
+| Annotation | Description
 
-a|[source,yaml]
-----
-annotations:
-  cpu-c-states.crio.io: "disable"
-  cpu-freq-governor.crio.io: "<governor>"
-----
-|Provides the best performance for a pod by disabling C-states and specifying the governor type for CPU scaling. The `performance` governor is recommended for high priority workloads.
-|====
+| `cpu-c-states.crio.io:`
+
+| This annotation allows you to `enable` or `disable` C-states for each CPU. Additionally, you can also specify a maximum latency in microseconds for the C-states. For example enable C-states with a maximum latency of 10 microseconds with the setting `cpu-c-states.crio.io: "max_latency:10"`. Set the value to `disable` to provide the best performance for a pod.
```

Comment:
Not sure I like "additionally" here as it could imply you can have multiple values. Maybe "Alternatively" would be better?